src/react-components/rtc-debug-panel.js

```diff
@@ -0,0 +1,1011 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import classNames from "classnames";
+import { AreaChart, Area, ResponsiveContainer } from "recharts";
+import { FormattedMessage } from "react-intl";
+import styles from "../assets/stylesheets/rtc-debug-panel.scss";
+
+const TransportType = {
+  SEND: "send_transport",
+  RECEIVE: "receive_transport"
+};
+
+const StatsType = {
+  INBOUND_RTP: "inbound-rtp",
+  OUTBOUND_RTP: "outbound-rtp",
+  LOCAL_CANDIDATE: "local-candidate",
+  REMOTE_CANDIDATE: "remote-candidate",
+  CANDIDATE_PAIR: "candidate-pair"
+};
+
+const STATS_REFRESH_TIME = 500;
+const PRODUCERS_KEY = "producers";
+const CONSUMERS_KEY = "consumers";
+const MEDIASOUP_DOC_BASE_URL = "https://mediasoup.org/documentation/v3/libmediasoupclient/api/";
+const MDN_DOC_BASE_URL = "https://developer.mozilla.org/en-US/docs/Web/API/";
+
+export class Prop extends Component {
+  static propTypes = {
+    propKey: PropTypes.string,
+    propValue: PropTypes.string
+  };
+
+  render() {
+    return (
+      <p className={classNames(styles.rtcTileText)}>
+        {`${this.props.propKey}: `}
+        <span className={classNames(styles.rtcValueText)}>{`${this.props.propValue}`}</span>
+      </p>
+    );
+  }
+}
+
+export class Button extends Component {
```

Comment:
I don't think you need to define a Button component in this file since the Button component in react-components/input/Button.js has what you need or can be modified to what you need.
If you don't think so, then I'd make a new button component in that folder.


```javascript
// import Button from ect. ect.

// the className prop should override OR you update the Button component if props.className exists, use props.className
<Button disabled={true/false} onClick={func} className={classNames(styles.rtcPanelButton)} >
       <FormattedMessage id={id} />
</Button>
```

Reply:
Refactored after our talk.